### PR TITLE
Feature 665 - Export repeated data in CSVs

### PR DESCRIPTION
**Pull request by kendrickt**
### Description
Update the CSV exporter to handle repeated answers.

NOTE: the CSV will not include answers to enumerator questions. The entity names picked by the applicant shouldn't matter to the admins.

### Screenshot
image

### Issue(s)
Progress towards #665

### CODE REVIEW

**Review by ndmckinley — APPROVED** [review]
Looks okay to me, though there's a lot in here I don't know that much about!  You might wanna wait for another approval, depending on your confidence level.

**Comment by kendrickt** [comment]
My confidence is probably at an all time low due to prod issues, but this doesn't touch storage, and I like my CSV picture and the tests I added.

**Pull request merged**
